grpc/tests/src/test/step.rs

```diff
@@ -85,41 +74,33 @@ fn should_step() {
         .with_parent_state_hash(builder.get_post_state_hash())
         .with_protocol_version(ProtocolVersion::V1_0_0)
         .with_slash_item(SlashItem::new(ACCOUNT_1_PK))
-        // TODO: currently, it is necessary to add a reward item for account_1 due to the genesis
-        // validator slashing immunity bug; when that bug is fixed, the below line adding
-        // reward item for ACCOUNT_1_PK and this comment should be removed.
         .with_reward_item(RewardItem::new(ACCOUNT_1_PK, 100000))
         .with_reward_item(RewardItem::new(ACCOUNT_2_PK, 100000))
         .build();
 
     let auction_hash = builder.get_auction_contract_hash();
 
-    let bid_purse_key = get_named_key(&mut builder, auction_hash, BID_PURSES_KEY);
     let reward_purse_key = get_named_key(&mut builder, auction_hash, VALIDATOR_REWARD_PURSE)
         .into_uref()
         .expect("should be uref");
-    let auction_seigniorage_key = get_named_key(
-        &mut builder,
-        auction_hash,
-        SEIGNIORAGE_RECIPIENTS_SNAPSHOT_KEY,
-    );
 
     let before_balance = builder.get_purse_balance(reward_purse_key);
     let before_auction_seigniorage: SeigniorageRecipientsSnapshot =
-        get_value(&mut builder, auction_seigniorage_key);
+        builder.get_value(auction_hash, SEIGNIORAGE_RECIPIENTS_SNAPSHOT_KEY);
 
-    // TODO: this is currently not possible due to a bug in the slashing logic
-    //       the genesis validators are not slashable due to an oversight in initialization.
-    // let _bid_purses_before_slashing: BidPurses = get_value(&mut builder, bid_purse_key);
-    // assert!(
-    //     bid_purses_before_slashing.contains_key(&ACCOUNT_1_PK),
-    //     "should contain slashed validator)"
-    // );
+    let bids_before_slashing: Bids = builder.get_value(auction_hash, BIDS_KEY);
+    assert!(bids_before_slashing.contains_key(&ACCOUNT_1_PK));
```

Comment:
Asserts should have a comment indicating what the failure means, particularly when there are more than one asserts in the same test.

Reply:
Done